thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/controller.js

```diff
@@ -399,27 +412,27 @@ export default Controller.extend({
   }),
 
   /**
-   * Update feedback status on any anomaly
-   * @method updateAnomalyFeedback
-   * @param {Number} anomalyId - the id of the anomaly to update
-   * @param {String} feedbackType - key for feedback type
-   * @return {Ember.RSVP.Promise}
-   */
-  updateAnomalyFeedback(anomalyId, feedbackType) {
-    const url = `/anomalies/updateFeedback/${anomalyId}`;
-    const data = { feedbackType, comment: '' };
-    return fetch(url, postProps(data)).then((res) => checkStatus(res, 'post'));
-  },
-
-  /**
-   * Fetch a single anomaly record for verification
-   * @method verifyAnomalyFeedback
-   * @param {Number} anomalyId
+   * Reset all time range options and activate the selected one
+   * @method newTimeRangeOptions
+   * @param {String} activeKey - label for currently active time range
    * @return {undefined}
    */
-  verifyAnomalyFeedback(anomalyId) {
-    const anomalyUrl = this.get('anomalyDataUrl') + anomalyId;
-    return fetch(anomalyUrl).then(checkStatus);
+  newTimeRangeOptions(activeKey) {
+    const timeRangeOptions = this.get('timeRangeOptions');
+    const newOptions = timeRangeOptions.map((range) => {
+      return {
+        name: range.name,
+        value: range.value,
+        isActive: false
+      };
+    });
+    const foundRangeOption = newOptions.find((range) => range.value === activeKey);
+
+    if (foundRangeOption) {
+      foundRangeOption.isActive = true;
+    }
+
+    return newOptions;
```

Comment:
Every new line above 435 should be removed (merge resolution fix)

Reply:
DONE